ios/FluentUI.Demo/FluentUIDemoTests/ActivityIndicatorTest.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import XCTest
+
+class ActivityIndicatorTest: XCTestCase {
+    let app = XCUIApplication()
+
+    override func setUpWithError() throws {
+        try super.setUpWithError()
+        continueAfterFailure = false
+        app.launch()
+
+        let onHomePage = app.staticTexts["FluentUI DEV"].exists
+        let onDifferentControlPage = !onHomePage && !app.staticTexts["ActivityIndicator"].exists
+        let activityIndicatorCell = app.staticTexts["ActivityIndicator"]
+        let backButton = app.buttons["FluentUI DEV"].exists ? app.buttons["FluentUI DEV"] : app.buttons["Dismiss"]
+
+        if onHomePage {
+            activityIndicatorCell.tap()
+        } else if onDifferentControlPage {
+            backButton.tap()
+            activityIndicatorCell.tap()
+        }
```

Comment:
Unfortunately this won't work because `onHomePage` is set before we tap the back button (it will still be false when we reach the if statement) so would it be cleaner to reset the variables or leave as is?

Reply:
Oh I see now - what you have now is fine!